pages/404.tsx

```diff
@@ -0,0 +1,23 @@
+import SignDirectionIcon from 'mdi-react/SignDirectionIcon'
+import * as React from 'react'
+import Layout from '../components/Layout'
+
+export class NotFoundPage extends React.Component<any, {}> {
```

Comment:
Small fix: because we already have `export default` below.

```suggestion
class NotFoundPage extends React.Component<any, {}> {
```

Idea: this could optionally be a function component (`React.FunctionComponent`)